{"name": "test02_mis_search", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <scripts.test02_mis.TestMis object at 0x0000019BEF52DCC0>\n\n    def test02_mis_search(self):\n        # 1. 调用查询文章接口\n        r = self.mis.api_mis_search()\n        print(\"查询文章成功后：\", r.json())\n        try:\n            # 2. 断言\n>           Tool.common_assert(r, status_code=200)\n\nscripts\\test02_mis.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ncls = <class 'tool.tool.Tool'>, response = <Response [403]>, status_code = 200\n\n    @classmethod\n    def common_assert(cls, response, status_code=201):\n        log.info(\"正在调用公共断言方法\")\n        # 断言状态码\n>       assert status_code == response.status_code\nE       AssertionError\n\ntool\\tool.py:23: AssertionError"}, "attachments": [{"name": "log", "source": "ec669138-41ef-4512-8749-cbf4bc517683-attachment.txt", "type": "text/plain"}], "start": 1612271258423, "stop": 1612271258517, "uuid": "b83c615d-ac31-4c20-9b25-0138dd9a7f9a", "historyId": "3f73d307b052a1555ac345fb66f890ec", "testCaseId": "717bcced00d6f137cd9d7d4949eeb939", "fullName": "scripts.test02_mis.TestMis#test02_mis_search", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test02_mis"}, {"name": "subSuite", "value": "TestMis"}, {"name": "host", "value": "DESKTOP-0Q83GN5"}, {"name": "thread", "value": "6236-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test02_mis"}]}